src/Microsoft.ML.FastTree/TreeEnsembleFeaturizer.cs

```diff
@@ -397,7 +397,7 @@ private void EnsureCachedPosition()
 
             public Func<int, bool> GetDependencies(Func<int, bool> predicate)
             {
-                for (int i = 0; i < Schema.ColumnCount; i++)
+                for (int i = 0; i < OutputSchema.ColumnCount; i++)
```

Comment:
>OutputSchema [](start = 36, length = 12)

InputSchema?

Reply:
I'm actually confused, and not sure which one is right.
Part of me thinks what if you create new row based on old one, your dependencies would be only on previous rows, which is InputSchema.


---
In reply to: [237959560](https://github.com/dotnet/machinelearning/pull/1759#discussion_r237959560) [](ancestors = 237959560)